src/workers/notifications/warmIntroNotification.ts

```diff
@@ -25,6 +26,19 @@ export const warmIntroNotification: TypedNotificationWorker<'gondul.v1.warm-intr
         return;
       }
 
+      await con.getRepository(OpportunityMatch).upsert(
+        {
+          userId,
+          opportunityId,
+          applicationRank: () =>
+            `applicationRank || '${JSON.stringify({ warmIntro: description })}'`,
```

Comment:
if warmIntro can contain quotes and stuff best to do it with parameters, like I did with opportunity https://github.com/dailydotdev/daily-api/blob/dd7ecee337d5502c87bb98dd72b6e8f6b65319ac/src/schema/opportunity.ts#L709